lib/simbricks/nicbm/nicbm.cc

```diff
@@ -297,8 +303,11 @@ void Runner::H2DRead(volatile struct SimbricksProtoPcieH2DRead *read) {
 #ifdef DEBUG_NICBM
   uint64_t dbg_val = 0;
   memcpy(&dbg_val, (const void *)rc->data, read->len <= 8 ? read->len : 8);
-  printf("main_time = %lu: nicbm: read(off=0x%lx, len=%u, val=0x%lx)\n",
-         main_time_, read->offset, read->len, dbg_val);
+  auto offset = read->offset;
+  auto len = read->len;
```

Comment:
What's your reason for introducing the additional variables?

Reply:
`cannot bind packed field 'read->SimbricksProtoPcieH2DRead::offset' to 'volatile long unsigned int&'`